sdk/ai/ai-projects/src/agents/traceUtility.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { TracingSpan } from "@azure/core-tracing";
+import { AgentsApiResponseFormat, AgentsApiResponseFormatOption, MessageContent, ThreadMessage, ThreadMessageOptions } from "./inputOutputs.js";
+import { TracingAttributes, TracingUtility } from "../tracing.js";
+import { getTelemetryOptions } from "../telemetry/telemetry.js";
+
+/**
+ * Adds a message event to the span.
+ * @param span - The span to add the event to.
+ * @param messageAttributes - The attributes of the message event.
+ */
+export function addMessageEvent(span: Omit<TracingSpan, "end">, messageAttributes: ThreadMessageOptions | ThreadMessage,): void {
+
+    const eventBody: Record<string, unknown> = {};
+    const telemetryOptions = getTelemetryOptions()
```

Comment:
Are these options the consumer is going to pass in? How will they configure these?

Reply:
They can configure via the Telemetry API that is exposed via client. I have it used in the tests.